cosmic-core/systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -1150,7 +1140,7 @@ def process(self):
         fwd = CsForwardingRules("forwardingrules", self.config)
         fwd.process()
 
-        # Strongswan is included in the systemvm template 17.3.12 and newer
+        # Strongswan is included in the systemvm template 17.3.13 and newer
         if get_systemvm_version() > 170312:
```

Comment:
@borisroman because the code `>17.3.12` and the comment were not in sync. Version `17.3.12` still has OpenSwan.